src/main/java/seedu/lifeasier/storage/FileCommand.java

```diff
@@ -74,4 +75,26 @@ public LocalDateTime convertToLocalDateTime(String dateTimeInformation) {
         return taskDateTime;
     }
 
+    /**
+     * Counts the number of save delimiters found in the read save file line.
+     *
+     * @param string String with which to count the number of save delimiters.
+     * @param baseDelimiterCount Base value to compare with.
+     * @return true when the count in given string matches baseDelimiterCount.
+     */
+    public boolean checkForDelimiterCount(String string, int baseDelimiterCount) {
+        boolean isStillContainingDelimiter = true;
+        int delimiterCount = 0;
+
+        while (isStillContainingDelimiter) {
+            isStillContainingDelimiter = string.contains(SAVE_DELIMITER);
+            if (isStillContainingDelimiter) {
+                delimiterCount++;
+                string = string.replaceFirst(SAVE_DELIMITER, BLANK_STRING);
```

Comment:
Could this be refactored into a do while loop?

Reply:
Alright, noted, will change